scss/header.scss

```diff
@@ -0,0 +1,124 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans&display=swap");
+
+.header {
+  background-color: aliceblue;
+  width: 100%;
+}
+
+.logo {
+  margin-left: 7.96%;
+  margin-top: 1%;
+  margin-bottom: 2%;
+}
+
+.img-hapo {
+  width: 100%;
+}
+
+.navbar {
+  background-color: aliceblue;
+}
+
+.navbar a:hover {
+  border-radius: 17px;
+  color: azure;
+  background: #b2d235;
+}
+
+.collapse {
+  position: absolute;
+  top: 30%;
+  right: 4%;
+}
+
+.nav-link {
+  font-family: "Open Sans", sans-serif;
```

Comment:
cái này set toàn bộ font-family cho thẻ body đi, các thẻ khác không phải set nữa